packages/shared-components/package.json

```diff
@@ -9,6 +9,10 @@
     "dist",
     "index.css"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/windicss/doc-utils"
```

Comment:
I use one line to fix the problem but takes me half hour to find where the code is. :(

Reply:
Sorry... you can ask me in windi's discord server next time : )